src/services/safes.rs

```diff
@@ -18,7 +19,13 @@ pub async fn get_safe_info_ex(
 ) -> ApiResult<SafeState> {
     let info_provider = DefaultInfoProvider::new(chain_id, context);
     let safe_info = info_provider.safe_info(safe_address).await?;
-    let safe_info_ex = safe_info.to_safe_info_ex(&info_provider).await;
+    let master_copy_url = core_uri!(info_provider, "/v1/about/master-copies/")?;
+    let supported_master_copies = request_master_copies(&context, master_copy_url)
+        .await
+        .unwrap_or(vec![]);
```

Comment:
Why don't we just fail?

Reply:
So that in the case the case we can't fetch the master copies for whatever reason, we still return the rest of the `SafeInfo` object. In the case of an empty vector, we would just return `Unknown` up-to-date state.

Reply:
hmmm ok, not sure if necessary, but I would comment this.

I think in general we should comment more on when we just fail and when do we use a fallback and why.